src/sysext/sysext.c

```diff
@@ -882,16 +901,17 @@ static int verb_help(int argc, char **argv, void *userdata) {
         _cleanup_free_ char *link = NULL;
         int r;
 
-        r = terminal_urlify_man("systemd-sysext", "1", &link);
+        r = terminal_urlify_man(directory_name(), "1", &link);
```

Comment:
hmm, `directory_name()` is  abit of a misnomer if you use it here too. maybe rename `directory_name()` to `systemd_syscfg_or_sysext()` to mirror `syscfg_or_sysext()`